test/conference.js

```diff
@@ -128,13 +128,13 @@ contract('Conference', function(accounts) {
       var meta;
       Conference.new().then(function(_meta) {
         meta = _meta;
-        beforeContractBalance = web3.eth.getBalance(meta.address);
+        beforeContractBalance = web3.eth.getBalance(meta.address).toNumber();
         return meta.register.sendTransaction(twitterHandle, {value:transaction});
       }).then(function() {
         return meta.totalBalance.call();
       })
       .then(function(value){
-        assert.equal(value.toString() - beforeContractBalance, transaction);
+        assert.equal(value..toNumber() - beforeContractBalance, transaction);
```

Comment:
I fixed on master. Do you know if there is a way to `git co $fork_branch` without cloning the fork repo? I was too lazy to test your branch before merging, hence I had to commit the fix directly on master.